movies/src/main/java/com/fundacionjala/movies/MovieChildren.java

```diff
@@ -0,0 +1,35 @@
+package com.fundacionjala.movies;
+
+/**
+ * Created by Charito on 6/10/2016.
+ */
+public class MovieChildren extends Movie {
+    public static final int CHILDREN = 2;
+
+    private static final double PRICE_RENT = 1.5;
+
+    private static final int delay = 3;
```

Comment:
this variable should be in uppercase.
I think this variable should be called DAYS_ALLOWED